src/Esfa.Vacancy.Api.Types/CreateApprenticeshipParameters.cs

```diff
@@ -78,5 +78,20 @@ public class CreateApprenticeshipParameters
         /// Provider site's unique reference number
         /// </summary>
         public int ProviderSiteEdsUrn { get; set; }
+
+        /// <summary>
+        /// Vacancy's contact name
```

Comment:
`Vacancy contact's name`

Just following the convention of the one used for email and phone number.